core/pom.xml

```diff
@@ -895,5 +895,16 @@ THE SOFTWARE.
         <findbugs.failOnError>true</findbugs.failOnError>
       </properties>
     </profile>
+    <profile>
+      <id>all-tests</id>
+      <activation>
+        <property>
+          <name>!test</name>
```

Comment:
should be true even when using -Dtest (because multiple forks and you can specify multiple tests)

Reply:
Just copying from `plugin-pom`. Skipped when `-Dtest=SomeTest` is specified in case this is a developer run.

Reply:
I possibly should have cleaned that up when I was there.  
problem is if you use `-Dtest=SomeTest,OtherTest -DforkCOunt=2` your output is garbage

Reply:
Yes, it is hard to read. Still better IMO than showing nothing.